src/bridge/cockpitrouter.c

```diff
@@ -703,8 +756,16 @@ process_open_dynamic_peer (CockpitRouter *self,
   if (config)
     json_object_unref (config);
 
+  /* Track peer in hash table */
+  g_hash_table_insert (self->channel_peers, (gpointer)g_strdup (channel), peer);
+
+  /* Handle dynamic peer channel redirect */
+  const gchar *redirect_channel;
+  cockpit_json_get_string (options, "redirect", NULL, &redirect_channel);
```

Comment:
Can you please add this to doc/authentication.md? This documents the JSON wire protocol, standard options (like this one), and the payload specific ones.

I'm not sure whether this should really be called "redirect", it feels more like an Unix pipe. But that's bikeshedding. :-)

Reply:
> I'm not sure whether this should really be called "redirect", it feels more like an Unix pipe. But that's bikeshedding. :-)

I didn't want to call it pipe because of possible confusion with `CockpitPipe`. (I guess it could be named "channel pipe", but  `CockpitChannelRedirect` already has its specific meaning.)

Reply:
> Can you please add this to doc/authentication.md? This documents the JSON wire protocol, standard options (like this one), and the payload specific ones.

I assume you meant doc/protocol.md (which contains the items you mentioned).

Reply:
Err yes, of course. :blush: